kbs/src/attestation/intel_trust_authority/mod.rs

```diff
@@ -76,19 +95,19 @@ impl Attest for IntelTrustAuthority {
         .to_string();
 
         // construct attest request data
-        let req_data = AttestReqData {
-            quote: evidence.quote,
-            runtime_data: STANDARD.encode(runtime_data),
+        let attest_req_body = match tee {
```

Comment:
The`att_url` could be included here: 
```suggestion
        let (req_data, att_url) = match tee {
```

with either `format!("{}/{ATTESTATION_API}", &self.config.base_url)` or `format!("{}/{ATTESTATION_API}/azure/tdxvm", &self.config.base_url)` returned